tests/testthat/test-unreachable_code_linter.R

```diff
@@ -597,6 +597,50 @@ test_that("function shorthand is handled", {
   )
 })
 
+test_that("Do not lint inline else after stop", {
+
+  expect_lint(
+    "if(x > 3L) stop() else x + 3",
```

Comment:
Irrelevant to the tests, but do you mind de-linting the tested code by adding a space between `if` and `(`?

Reply:
I have done it